docs/guides/mkdocs-old-versions.rst

```diff
@@ -0,0 +1,75 @@
+Keep Building Docs With Old Version Of MkDocs
+=============================================
+
+Recent changes on ``mkdocs`` forced us to `upgrade the default version installed`_ by Read the Docs and this may be a breaking change for your documentation.
+
+.. _upgrade the default version installed: https://github.com/rtfd/readthedocs.org/pull/4041
+
+You should check that your docs continue building in any of these cases:
+
+* your project doesn't have a ``requirements.txt`` file pinning ``mkdocs`` to a specific version
+* your project is using a custom theme
+* your project is using Markdown extensions
+
+In case your builds are failing because of a ``mkdocs`` issue,
+you may want to follow one of the following solutions depending on the case.
+
+
+Pin mkdocs to the previous version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before Read the Docs upgraded its default version installed, ``mkdocs==0.15.0`` was used.
+To make your project continue using this version you will need to create a ``requirements.txt`` file with this content::
+
+     # requirements.txt
+     mkdocs==0.15.0
+     mkdocs-bootstrap==0.1.1
+     mkdocs-bootswatch==0.1.0
+
+.. note::
+
+   Most of the breaking changes where introduced in ``mkdocs==0.17``,
+   so you may want to test only pinning ``mkdocs`` to ``mkdocs<0.17``
+   and check if your docs keep building successfully.
+
+
+Upgrade your custom theme to be compatible with newer mkdocs versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible that event your build passes your documentation doesn't look correct.
+This may be because the new ``mkdocs==0.17.3`` version installed by Read the Docs introduced some breaking changes on the structure of the theme.
+
+You should check the `mkdocs' Custom Theme documentation`_ to upgrade your custom theme and make it compatible with the new version.
+
+.. _mkdocs' Custom Theme documentation: https://www.mkdocs.org/user-guide/custom-themes/
+
+
+Upgrade how extension arguments are defined
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
I actually couldn't find exactly where this syntax changed. I tested building docs with the old method in 0.15 and they still didn't build. Maybe this was a change in a markdown parser? I'm not sure.

Reply:
Did you pin `Markdown` also to some version? I didn't, and I had to upgrade this.

Reply:
Got it. Maybe this changed underneath mkdocs.